frontend/public/components/factory/table-filters.ts

```diff
@@ -44,15 +44,35 @@ export const tableFilters: TableFilterMap = {
     return haystack.includes(needle);
   },
 
-  'alert-state': (filter, alert) => filter.selected.has(alertState(alert)),
+  'alert-list-label': (values, alert) => {
+    const labels = getLabelsAsString(alert, 'labels');
+    if (!values.all) {
+      return true;
+    }
+    return !!values.all.every((v) => labels.includes(v));
+  },
```

Comment:
Looks like `alert-list-label` and `alerting-rule-label` can be combined to make one `alerts-label` filter and use it. 
```suggestion
  'alerts': (values, alert) => {
    const labels = getLabelsAsString(alert, 'labels');
    if (!values.all) {
      return true;
    }
    return !!values.all.every((v) => labels.includes(v));
  },
```